### PR TITLE
Quick tests

### PR DESCRIPTION
**Pull request by bdemann**

### CODE REVIEW

File: tests/end_to_end/http_server/large_files/test/auto_tests.ts [hunk 1]
```diff
@@ -10,22 +10,28 @@ const autoGenAutoUploadSmallFileInfos: [number, Unit][] = [
...
     [10, 'KiB'],
-    [100, 'KiB'],
-    [1, 'MiB'],
-    [10, 'MiB'],
-    [100, 'MiB']
+    [100, 'KiB']
 ];
 
 const autoGenAutoUploadFileInfos: [number, Unit][] =
     process.env.AZLE_TEST_RUN_ON_RELEASE === 'true' ||
```
Comment (lastmjs) [inline tests/end_to_end/http_server/large_files/test/auto_tests.ts:21]: Prefix all env variables in test.yml

Comment (lastmjs) [inline tests/end_to_end/http_server/large_files/test/auto_tests.ts:21]: Rename to AZLE_TEST_RUN_ON_RELEASE_BRANCH_PR

Comment (lastmjs) [inline tests/end_to_end/http_server/large_files/test/auto_tests.ts:21]: Let's just use the regular test.yml variables, but prefixed with AZLE_. We should explicitly put in each case (release branch pr, main commit thing)

**Review by lastmjs — CHANGES_REQUESTED** [review]

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -38,24 +40,68 @@ jobs:
...
+                  tests/end_to_end/candid_rpc/class_syntax/stable_structures
+                  tests/end_to_end/candid_rpc/functional_syntax/bitcoin
+                  tests/end_to_end/candid_rpc/functional_syntax/composite_queries
+                  tests/end_to_end/candid_rpc/functional_syntax/cross_canister_calls
+                  tests/end_to_end/candid_rpc/functional_syntax/management_canister
+                  tests/end_to_end/candid_rpc/functional_syntax/stable_structures
+                  tests/end_to_end/http_server/autoreload
+                  ') }}"
+
+                  # Exclude RELEASE_TESTS if it's not a release pr
```
Comment (lastmjs) [inline .github/workflows/test.yml:78]: One if statement per branch/push type

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -132,27 +177,30 @@ jobs:
               shell: bash -l {0}
 
             - name: Start dfx without artificial delay
```
Comment (lastmjs) [inline .github/workflows/test.yml:179]: Start dfx with artificial delay 0

File: .github/workflows/test.yml [hunk 4]
```diff
@@ -132,27 +177,30 @@ jobs:
...
-                  IS_RELEASE_BRANCH_PR_BEFORE_MERGE_TO_MAIN=${{ env.IS_RELEASE_BRANCH_PR == 'true' && env.IS_MAIN_BRANCH == 'false' }}
-
-                  RUNS=5
-                  if $IS_MERGE_TO_MAIN_FROM_RELEASE; then
+                  RUNS=1
+                  if ${{ env.AZLE_IS_MAIN_BRANCH_MERGE_FROM_RELEASE_PUSH }}; then
                       RUNS=100
                   fi
-                  if $IS_RELEASE_BRANCH_PR_BEFORE_MERGE_TO_MAIN; then
+                  if ${{ env.AZLE_IS_MAIN_BRANCH_PUSH }}; then
```
Comment (lastmjs) [inline .github/workflows/test.yml:195]: Same order as definition

**Review by lastmjs — CHANGES_REQUESTED** [review]

File: .github/workflows/test.yml (hunk 2 above)
Comment (bdemann) [reply .github/workflows/test.yml:78]: done

File: .github/workflows/test.yml (hunk 3 above)
Comment (bdemann) [reply .github/workflows/test.yml:179]: done

File: .github/workflows/test.yml (hunk 4 above)
Comment (bdemann) [reply .github/workflows/test.yml:195]: done

**Review by lastmjs — CHANGES_REQUESTED** [review]
Add liney line breaks

**Review by lastmjs — APPROVED** [review]

**Pull request merged**